src/cli/commands/remove.js

```diff
@@ -41,13 +41,13 @@ export async function run(
       const registry = config.registries[registryName];
       const object = rootManifests[registryName].object;
 
-      for (const type of constants.DEPENDENCY_TYPES) {
```

Comment:
This construct was crashing node on ubuntu on windows.
I know it is not a good idea to patch Yarn because of a very niche environment but I could not find a nicer workaround